sweep-example--fibonacci-model/README.md

```diff
@@ -0,0 +1,92 @@
+# Fibonacci Model
+
+This is an example model. Here you should describe what the model does.
+
+This particular model generates a Fibonacci sequence and saves it to a file in such a way 
+to allow this Model to function with Sweep steps. e.g.:
```

Comment:
I think it would be clearer to move the bit talking about sweep steps here to a separate line/sentence, as it implies that the sequence below is related to Sweep steps